src/modules/db_operations/seed_trait_files.py

```diff
@@ -0,0 +1,96 @@
+from caendr.services.logger        import logger
+from caendr.services.cloud.secret  import get_secret
+
+from caendr.models.datastore       import TraitFile
+from caendr.models.error           import NotFoundError, NonUniqueEntity
+from caendr.models.status          import PublishStatus
+from caendr.services.cloud.sheets  import get_field_from_record
+from caendr.utils.data             import unique_id
+from caendr.utils.local_files      import LocalGoogleSheet
+
+
+# Get secret value(s)
+ANDERSEN_LAB_TRAIT_SHEET = get_secret(f'ANDERSEN_LAB_TRAIT_SHEET')
+
+
+
+def populate_andersenlab_trait_files():
+  '''
+    Create / update datastore entities for each record in the trait list Google Sheet.
+    Assumes the following:
```

Comment:
we should explicitly document that this function does not remove rows from datastore if a ROW is removed from the sheets after having been synced to datastore.